File: w3d5/Knights Travails/board.rb
class Board
   def initialize(pos)
        @grid = Array.new(8) {Array.new(8)} 
    end

    def grid
        @grid
    end

    def [](pos)
        row, col = pos
        @grid[pos]
    end

    def []=(pos, move) 
        row, col = pos
        @grid[pos] = move
    end

end

File: w8d5/tic-tac-toe/board.js
class Board {
    constructor() {
        this.board = new Array()
        for (let i = 0; i <3; i++) {
            this.board.push(["_", "_", "_"])
        }
    }
    print() {
        this.board.forEach (row => {
            console.log(row)
        } )
    }
} 
// const b = new Board()
// b.print()